front/app/hooks/useSomeFeatureFlags.ts

```diff
@@ -0,0 +1,40 @@
+import { useState, useEffect } from 'react';
+import {
+  currentAppConfigurationStream,
+  IAppConfiguration,
+  TAppConfigurationSetting,
+} from 'services/appConfiguration';
+import { get } from 'lodash-es';
+
+type Parameters = {
+  names: TAppConfigurationSetting[];
```

Comment:
But I copied this name `names` from `useFeatureFlag`. It has the `name` parameter. So, I made it similar, which is important I believe. Do you think I should rename the parameter in `useFeatureFlag` too? (the diff will be huge)

Reply:
In fact, you would also have to change it for FeatureFlag (component) and GetFeatureFlag too. You see here that we have similar type definitions in all 4 of them, so this would be a chance to use the same interface (or type) for all 4 off them.

I believe the name `name` is bad. From the outside, I can't tell what it means. I'd have to check the type definitions inside these files to understand, but it's also not super bad in that it's costing us much time or we're introducing hard-to-reverse tech debt. So you can leave it like this.